recipes/aimz/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "aimz" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/aimz-{{ version }}.tar.gz
+  sha256: bd2c4250224a26812a37268acbea9253b5ef522fbc3d264ec6894523fb67847e
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [py<311 or win]
```

```suggestion
build:
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
  number: 0
  noarch: python
```
This package should be `noarch` - we can set the `python_min` as shown above.  If the package fails on Windows because of missing dependencies, don't worry about that - we'll merge as is.